docs/docs/configuration/snapshots.md

```diff
@@ -21,103 +39,149 @@ The materialization also supports invalidating hard deletes. Check the [docs](ht
+11,New Mateo,2014-09-07T17:04:27
+12,New Julian,2000-02-04T11:48:30
+13,New Gabriel,2001-07-10T07:32:52
 ```
 
 </File>
 
-<File name='snapshots/model_snapshot_timestamp_strategy.sql'>
+
+<File name='snapshots/iceberg_snapshot_timestamp_strategy.sql'>
 
 ```sql
-{% snapshot model_snapshot_timestamp_strategy %}
-  {{
-      config(
+{% snapshot iceberg_snapshot_timestamp_strategy %}
+    {{ config(
         strategy='timestamp',
-        updated_at='refresh_timestamp',
-        unique_key='id'
-      )
-  }}
-
-  select * from {{ ref('model') }}
+        unique_key='id',
+        updated_at='some_date',
+        table_type='iceberg',
+    )}}
+    select * from {{ ref(var('seed_name', 'base')) }}
 {% endsnapshot %}
 
 ```
 
 </File>
 
-<File name='snapshots/model_snapshot_timestamp_strategy_hard_deletes.sql'>
+<File name='snapshots/hive_snapshot_timestamp_strategy_hard_deletes.sql'>
 
 ```sql
-{% snapshot model_snapshot_timestamp_strategy_hard_deletes %}
-  {{
-      config(
-        unique_key='id',
+{% snapshot hive_snapshot_timestamp_strategy_hard_deletes %}
+    {{ config(
         strategy='timestamp',
-        updated_at='refresh_timestamp',
+        unique_key='id',
+        updated_at='some_date',
+        table_type='hive',
         invalidate_hard_deletes=True,
-      )
-  }}
-  select * from {{ ref('model') }}
+    )}}
+    select * from {{ ref(var('seed_name', 'base')) }}
 {% endsnapshot %}
 ```
 
 </File>
 
-<File name='snapshots/model_snapshot_check_strategy.sql'>
+<File name='snapshots/iceberg_snapshot_check_column_strategy.sql'>
 
 ```sql
-{% snapshot model_snapshot_check_strategy %}
-  {{
-      config(
-        unique_key='id',
+{% snapshot iceberg_snapshot_check_column_strategy %}
+    {{ config(
+        check_cols=['name'], 
+        unique_key='id', 
+        strategy='check',
+        table_type='iceberg',
+    ) }}
+    select * from {{ ref(var('seed_name', 'base')) }}
+{% endsnapshot %}
+```
+
+</File>
+
+<File name='snapshots/hive_snapshot_check_all_columns_strategy.sql'>
+
+```sql
+{% snapshot hive_snapshot_check_all_columns_strategy %}
+    {{ config(
+        check_cols='all', 
+        unique_key='id', 
         strategy='check',
-        check_cols=['series_reference','data_value']
-      )
-  }}
-  select * from {{ ref('model') }}
+        table_type='hive',
+    ) }}
+    select * from {{ ref(var('seed_name', 'base')) }}
 {% endsnapshot %}
 ```
 
 </File>
 
+Now you can test these different snapshot strategies:
+
+```shell
+# Load seeds
+dbt seed
+
+# Run initial snapshot
+dbt snapshot --select hive_snapshot_check_all_columns_strategy
+
+# Run snapshot again, on changed source data
+dbt snapshot --vars "seed_name: changed"
+```
+
 :::caution
 Snapshots do not support dropping columns from the source table. If you drop a column, make sure to drop the column from the snapshot as well. Another workaround is to `NULL` the column in the snapshot definition to preserve history
 :::
+
+## Version 1.5 migration guide
```

Comment:
Maybe you can create a new doc in `docs/docs/UPGRADE_1.5.md`  and link to that page from the :::caution::: blocks? To keep the main snapshots page cleaner?